### PR TITLE
Add Derivative: Cangjie

### PR DESCRIPTION
**Pull request by BinanceNFT**
## Cangjie

Etherscan link: https://etherscan.io/address/0x6Ce13E96CBaEF984496fBA4FDE0D3D48C8e30c25

Website: https://cangjie.art

Twitter: https://twitter.com/CangjieNFT

OpenSea: https://opensea.io/collection/cangjie

### CODE REVIEW

File: utils/lists.ts [hunk 1]
```diff
@@ -227,6 +227,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Randomized adventurer meals",
     url: "https://etherscan.io/address/0x615a610649e656485d9baf0ebe525496d7b78e24",
   },
+  {
+    name: "Cangjie",
+    description: "Cangjie(仓颉) was the creator of Chinese characters. The Cangjie NFTs are Chinese characters generated with Taiji, Wuxing, Yi Jing, and more.",
```
Comment (ravindranrahul) [inline utils/lists.ts:232]: Can you please make the description shorter? so that it fits in one line like others

**Review by ravindranrahul — CHANGES_REQUESTED** [review]

File: utils/lists.ts (hunk 1 above)
Comment (CangjieNFT) [reply utils/lists.ts]: I have fixed the description. Please re-check it. Thank you.

File: utils/lists.ts [hunk 2]
```diff
@@ -227,6 +227,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Randomized adventurer meals",
     url: "https://etherscan.io/address/0x615a610649e656485d9baf0ebe525496d7b78e24",
   },
+  {
+    name: "Cangjie(倉頡)",
+    description: "The first Chinese characters Loot project with Yinyang, Wuxing and Zhouyi ☯.",
```
Comment (ravindranrahul) [inline utils/lists.ts:232]: Length looks good. Please remove the trailing period in description and that should be it

**Review by ravindranrahul — CHANGES_REQUESTED** [review]

File: utils/lists.ts (hunk 2 above)
Comment (CangjieNFT) [reply utils/lists.ts]: Done. Thanks for your patient.

**Comment by ravindranrahul** [comment]
Looks good! Thank you

**Pull request merged**